spec/features/event_types_spec.rb

```diff
@@ -32,7 +32,7 @@
       within('table#event_types') do
         expect(page.has_content?('Party')).to be true
         expect(page.has_content?('13042')).to be true
-        expect(page.has_content?('#e4e4e4')).to be true
+        expect(page.has_content?('#E4E4E4')).to be true
```

Comment:
You could do an [interactive rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i):

```
git rebase -i HEAD~3
```